File: app/src/main/java/com/doordash/binding/binder/RestaurantBinder.java
package com.doordash.binding.binder;

import com.doordash.binding.binder.common.ConditionalDataBinder;
import com.doordash.binding.viewmodel.StoreModel;


public class RestaurantBinder extends ConditionalDataBinder<StoreModel>
{
    public RestaurantBinder(int bindingVariable, int layoutId)
    {
        super(bindingVariable, layoutId);
    }

    @Override
    public boolean canHandle(StoreModel model) {
        return true;
    }


}

File: app/src/main/java/com/doordash/binding/binder/RestaurantBinder.kt
package com.doordash.binding.binder

import com.doordash.bean.Store
import com.doordash.binding.binder.common.ConditionalDataBinder

class RestaurantBinder(bindingVariable: Int, layoutId: Int) : ConditionalDataBinder<Store>(bindingVariable, layoutId) {


    override fun canHandle(model: Store): Boolean {
      return true
    }
}